### ISSUE TITLE
[zh-hans] Translation Fix in setting page: Wrapping Strategy.

### ISSUE THREAD

**Issue by neville-studio** [microsoft/vscode-loc#1764]
The English `Controls the algorithm that computes wrapping points. Note that when in accessibility mode, advanced will be used for the best experience` should be translated to `控制计算换行位置的算法。注意，在辅助模式下，高级模式将会提供最佳体验` is better rather than `控制计算包装点的算法。请注意，在辅助功能模式下，高级版将用于提供最佳体验`

**Especially note that the word `wrapping` should be translated to `换行` rather than `包装`**

package id: ms-ceintl.vscode-language-pack-zh-hans
version: 1.96.2024121109
last update: 2024-12-23, 22:29:52

**Comment by csigs** [comment]
Thank you for your feedback. We will review the issue (ICX ADO Tracking Reference: 1023003).

**Comment by cristianosuzuki77** [comment]
Internally tracking via ADO#1145316
